pallets/polkadex-ido/src/lib.rs

```diff
@@ -506,122 +225,99 @@ pub mod pallet {
+            token_a_price_per_token_b: BalanceOf<T>,
         ) -> DispatchResult {
             let team: T::AccountId = ensure_signed(origin)?;
-
-            let (token_a, mintnew) = if let Some(token_a) = token_a {
-                //TODO check if funder have the token_a available and reserve them.
-                (token_a, false)
-            } else {
-                (Self::create_random_token()?, true)
-                //TODO  mint the new token and reserve them.
-                //TODO Make sure we mint the random tokens in an acceptable range.
-            };
-
-            let current_block_no = <frame_system::Pallet<T>>::block_number();
-            let vote_end_block = match <VotingPeriod<T>>::try_get() {
-                Ok(voting_period) => voting_period.saturating_add(current_block_no),
-                Err(_) => T::DefaultVotingPeriod::get().saturating_add(current_block_no)
-            };
-            ensure!(token_a.ne(&token_b), <Error<T>>::TokenAEqTokenB);
-
-            let start_block = vote_end_block.clone().saturating_add(1_u128.saturated_into());
-            let close_round_block = vote_end_block.saturating_add(funding_period);
-            // Ensures that
-            let token_a_priceper_token_b_perquintill = Perbill::from_rational(token_a_priceper_token_b, 1_000_000_000_000_u128.saturated_into());
-
+            //TODO check if funder have the token_a available and reserve them.
             // CID len must be less than or equal to 100
             ensure!(cid.len() <= 100, <Error<T>>::CidReachedMaxSize);
-            ensure!(!token_a_priceper_token_b_perquintill.is_zero(), <Error<T>>::PricePerTokenCantBeZero);
             ensure!(min_allocation <= max_allocation, <Error<T>>::MinAllocationMustBeEqualOrLessThanMaxAllocation);
-            ensure!(start_block < close_round_block, <Error<T>>::StartBlockMustBeLessThanEndblock);
-            ensure!(vote_end_block < start_block, <Error<T>>::StartBlockMustBeGreaterThanVotingPeriod);
             ensure!(vesting_per_block > Zero::zero(), <Error<T>>::VestingPerBlockMustGreaterThanZero);
-
-
-            // Mint random token if user selects none: TODO: Remove in production, only for beta testes
-            ///TODO check if an old or new token again here and only mint the new
-
+            let current_block_no = <frame_system::Pallet<T>>::block_number();
+            let start_block = current_block_no.clone().saturating_add(1_u128.saturated_into());
+            let close_round_block = current_block_no.saturating_add(funding_period);
+            let token_a_price_per_token_b_perquintill = Perbill::from_rational(token_a_price_per_token_b, 1_000_000_000_000_u128.saturated_into());
+            ensure!(!token_a_price_per_token_b_perquintill.is_zero(), <Error<T>>::PricePerTokenCantBeZero);
+            ensure!(start_block < close_round_block, <Error<T>>::StartBlockMustBeLessThanEndblock);
             let vesting_period: u32 = (amount / vesting_per_block).saturated_into();
```

Comment:
Don't use div symbol anywhere in runtime

Reply:
Removing vesting period for now in this code, as it will not be used here and instead will be included in another PR